hera_mc/geo_location.py

```diff
@@ -208,7 +212,7 @@ def is_in_connections(args, station_name, check_if_active=False):
     ------------
     args:  needed arguments to open database and set date/time
     station_name:  string name of station
-    check_if_active:  boolean flag to check if active
+    check_if_active:  either False to not check/care if active, or astropy Time to check
```

Comment:
the name of this parameter is confusing and the mixed type (boolean or time) is weird. Why not use something that has time in its name and if it's none, not care if it's active?